source/geometry/src/GateCylindricalPETSystem.cc

```diff
@@ -49,6 +49,11 @@ GateCylindricalPETSystem::GateCylindricalPETSystem(const G4String& itsName)
   new GateBoxComponent("layer1",crystalComponent,this);
   new GateBoxComponent("layer2",crystalComponent,this);
   new GateBoxComponent("layer3",crystalComponent,this);
+  new GateBoxComponent("layer4",crystalComponent,this);
+  new GateBoxComponent("layer5",crystalComponent,this);
+  new GateBoxComponent("layer6",crystalComponent,this);
+  new GateBoxComponent("layer7",crystalComponent,this);
+  new GateBoxComponent("layer8",crystalComponent,this);
```

Comment:
Why do we need to create this? This looks strange.

Reply:
By default, GATE allows to build only 4 layers and they are hardcoded. So I added 5 more to be able to build the most extreme case scanner.